### ISSUE TITLE
Concept: Html.lazy

### ISSUE THREAD

**Issue by ceddlyburge** [exercism/elm#587]
Not sure how easy / possible this will be to do, so leaving this here as a placeholder for now.

There is quite a bit of confusion with Html.lazy though, which shows up on the Elm slack, so would be good to teach it if we can

**Comment by mpizenberg** [comment]
Yep, this and Html.Keyed often people misuse them or misinterpret their usefulness.
Also quite difficult to teach as they both rely on the VDOM implementation that we don't exercise yet. Would require a way of building html (and testing) pages in exercism.
